.github/workflows/ci.yml

```diff
@@ -115,7 +115,7 @@ jobs:
             --http-status-ignore 405,429,503,999 \
             --url-ignore "/.*localhost.*/","/.*vimeo\.com.*/","/.*gitter\.im.*/","/.*drmaa\.org.*/","/.*slides.html#.*/" \
             --url-swap "github.com/galaxyproject/training-material/tree/master:github.com/${GITHUB_REPOSITORY}/tree/${GITHUB_HEAD_REF}" \
-            --file-ignore "/.*\/files\/.*/","/.*\/node_modules\/.*/" \
+            --file-ignore "/.*\/files\/.*/","/.*\/node_modules\/.*/","/\/tutorials\/.*\/docker\//" \
```

Comment:
Could you include this in the Makefile as well, in the root of the repository?

Reply:
_check-html and _check-html-internal are done - let me know about check-links-gh-pages - not sure what that does - looks like it is for a specific branch? Huh?